flintrock/config.yaml.template

```diff
@@ -3,6 +3,9 @@ services:
     version: 1.6.1
     # git-commit: latest  # if not 'latest', provide a full commit SHA; e.g. d6dc12ef0146ae409834c78737c116050961f350
     # git-repository:  # optional; defaults to https://github.com/apache/spark
+    # optional; defaults to download from a dynamically selected Apache mirror
+    # must contain a {v} template corresponding to the version; must be a .tar.gz file
+    # download-source: "https://www.example.com/files/spark/{v}/spark-{v}.tar.gz"
```

Comment:
Perhaps something like this?

```
# optional; defaults to download from official Spark S3 bucket
#   - must contain a {v} template corresponding to the version
#   - Spark must be pre-built
#   - must be a .tar.gz file
# download-source: "https://www.example.com/files/spark/{v}/spark-{v}.tar.gz"
```

And then we should update the matching comment for the Hadoop download source to follow similar formatting.